apps/web/utils/actions/group.ts

```
@@ -12,32 +13,116 @@ import {
 import { getGmailClient, getGmailAccessToken } from "@/utils/gmail/client";
-import { GroupItemType } from "@prisma/client";
-import type { ServerActionResponse } from "@/utils/error";
+import { GroupItemType, Prisma, User } from "@prisma/client";
+import { captureException, type ServerActionResponse } from "@/utils/error";
 import {
   NEWSLETTER_GROUP_ID,
   RECEIPT_GROUP_ID,
 } from "@/app/(app)/automation/create/examples";
 import { GroupName } from "@/utils/config";
+import { aiGenerateGroupItems } from "@/utils/ai/group/create-group";
+import { UserAIFields } from "@/utils/llms/types";
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the transpilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)